build-system/tasks/karma.conf.js

```diff
@@ -304,7 +304,7 @@ module.exports = {
   // TODO(rsimha): Reduce this number once keepalives are implemented by
   // karma-sauce-launcher.
   // See https://github.com/karma-runner/karma-sauce-launcher/pull/161.
-  browserDisconnectTimeout: 15 * 60 * 1000,
+  browserDisconnectTimeout: 2 * 60 * 1000,
```

Comment:
For posterity, can you add a note to the description explaining what this setting does and why it was changed?

Reply:
Done.